pom.xml

```diff
@@ -61,10 +61,11 @@
 
     <properties>
         <!-- compile, runtime and provided dependency versions -->
-        <apache.avro.version>1.11.4</apache.avro.version>
+        <apache.avro.version>1.12.0</apache.avro.version>
         <apache.commons.text.version>1.12.0</apache.commons.text.version>
         <apache.commons.compress.version>1.26.2</apache.commons.compress.version>
         <apache.kafka.version>3.8.0</apache.kafka.version>
+        <apicurio.avro.version>3.0.0.M4</apicurio.avro.version>
```

Comment:
Do we want to use the milestone release already?
What are the chances on library changes and possible compatibility issues with Apicurio Registry 2.x